File: fibonacci_series.cpp
#include <iostream>

using namespace std;

int main(){
    int t = 1, t_1 = 1;
    int num, x;

    cout << " Enter Number: ";
    cin >> num;
    cout << endl;
    cout << t << "\t" << t_1  << "\t";
    for(int i=1; i<=num-2; i++){
        x = t + t_1;
        cout << x << "\t";
        t_1 = t;
        t = x;
    }
    return 0;
}

File: math_algorithms/fibonacci_series/fibonacci_series.py
def fibonacci_series(num: int):
    t, t_1 = 1, 1
    print(t, '\t', t_1, end='\t')
    for i in range(1, num-1):
        x = t + t_1
        t_1 = t
        t = x
        print(x, end='\t')


num = int(input("Enter Number: "))

fibonacci_series(num)
